### PR TITLE
Change textdomain from mixed case to lowercase

### LINKED ISSUE #192: [Bug]: Translations require lowercase textdomain
### Contact Details

andy@thefragens.com

### What happened?

GlotPress exports files as lowercase, not mixed case. Consequently all the text domains need to be switched to lowercase for the translations to load. Also we no longer need to call `load_plugin_textdomain`.

### Version

0.5 (Default)

### OS

MacOS

### What browsers are you seeing the problem on?

Safari

### WordPress

6.7

### PHP

8.2

### Server

_No response_

### Database

_No response_

### Theme

_No response_

### Must-Use Plugins

_No response_

### Plugins

_No response_

### WordPress Site Health Output (optional)

_No response_

### Relevant debug log output

_No response_

### Screenshots/Live demo link

_No response_

### Additional Notes

_No response_

### Code of Conduct

- [X] I agree to follow the Code of Conduct.

### PR DESCRIPTION
**Pull request by afragen**
GlotPress exports files in lowercase. As the current textdomain is mixed case, this is problematic as the translation files won't load. Converting from mixed case textdomain to lowercase fixes this issue.

There is no longer a need to call `load_plugin_textdomain` as the translation files will be loaded JIT by core.

Fixes #192

### CODE REVIEW

**Comment by namithj** [comment]
We will also need to update the releases action to make sure it uses the plugin folder name in the lowercase and not the Capitalized repo name

**Comment by afragen** [comment]
Fixes #192

**Comment by afragen** [comment]
@namithj the release action already uses the repository name. And yes, it would  be easier if the repository name were  `aspireupdate` and not `AspireUpdate`. 😉

**Comment by afragen** [comment]
Hmm, no idea why I am unable to rename the POT to lowercase. It's lowercase locally.

File: includes/class-branding.php [hunk 1]
```diff
@@ -122,9 +122,9 @@ public function output_admin_notice() {
 			case 'update-core-network':
 				$message = sprintf(
 					/* translators: 1: The name of the plugin, 2: The documentation URL. */
-					__( 'Your WordPress, plugin, theme and translation updates are now powered by <strong>%1$s</strong>. <a href="%2$s">Learn more</a>', 'AspireUpdate' ),
-					'AspireUpdate',
-					__( 'https://docs.aspirepress.org/aspireupdate/', 'AspireUpdate' )
+					__( 'Your WordPress, plugin, theme and translation updates are now powered by <strong>%1$s</strong>. <a href="%2$s">Learn more</a>', 'aspireupdate' ),
+					'aspireupdate',
```
Comment (costdev) [inline includes/class-branding.php:126]: This one should remain -> It's the name of the plugin being inserted into `%1$s` in the `sprintf()` call.

Comment (afragen) [reply includes/class-branding.php]: fixed

**Review by ipajen — APPROVED** [review]
What I tested it worked fine. Just the slug needs to be updated to aspireupdate so the PCP warnings goes away,

**Pull request merged**